cinema/serializers.py

```diff
@@ -68,13 +87,56 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session")
```

Comment:
check please if that movie session returns data in this format instead of movie_session id
<img width="340" alt="image" src="https://github.com/mate-academy/py-tickets-orders/assets/61380547/fd583637-16ea-4094-951b-a8f312856243">